releasenotes/notes/fix-stop-words-strip-issue-22ce51306e7b91e4.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Ensure the leading whitespace in the generated text is preserved when using `stop_words` in the Hugging Face invocation layer of the PromptNode
```

```suggestion
    Ensure the leading whitespace in the generated text is preserved when using `stop_words` in the Hugging Face invocation layer of the PromptNode.
```
Sorry. Please add a full stop. :raised_hands:

---

It was my fault! :smile:

---

never mind 😊